pyproject.toml

```diff
@@ -49,25 +49,11 @@ pangeo = [
     "gcsfs>=2024.2",
 ]
 all = [
-    # viewer
-    "trame>=3.6",
-    "trame-vtk>=2.6",
-    "trame-vuetify>=2.4",
-    "geovista>=0.4",
-
-    # esgf
-    "intake-esgf>=2024.1",
-
-    # pangeo
-    "intake==0.7.0",  # latest has plugin errors
-    "intake-xarray>=0.7",
-    "intake-esm>=2023.11",
-    "s3fs>=2024.2",
-    "gcsfs>=2024.2",
```

Comment:
I think this change is outdated, so you can take this out. I believe @johnkit had mentioned that the recursive dependency definition doesn't work in some cases, so we should use the explicit list.